src/components/typography-control/index.js

```diff
@@ -107,7 +107,7 @@ const TypographyControl = props => {
 							{ label: __( 'Uppercase', i18n ), value: 'uppercase' },
 							{ label: __( 'Lowercase', i18n ), value: 'lowercase' },
 							{ label: __( 'Capitalize', i18n ), value: 'capitalize' },
-							{ label: __( 'Normal', i18n ), value: 'normal' },
+							{ label: __( 'Normal', i18n ), value: 'none' },
```

Comment:
Since there isn't a `normal` value in the first place, let's also change the label to `Normal`
```suggestion
							{ label: __( 'None', i18n ), value: 'none' },
```